### ISSUE TITLE
admin: subnet badge shows warning even when all routes are approved

### ISSUE THREAD

**Issue by charlieporth1** [tailscale/tailscale#4088]
### What is the issue?

React state not updating properly for multiple subnets in the admin web interface. (I am assuming you are using react for the web interface). 
![image](https://user-images.githubusercontent.com/13573364/157086138-3541d083-54fe-4c97-a80b-ec1dcc31fd90.png)

Toggling subnets in a specific order fixes that

### OS

Other

### OS version

Web

### Tailscale version

N/a

**Comment by bradfitz** [comment]
cc @tailscale/frontend

**Comment by rosszurowski** [comment]
Ah, I see what you mean. It looks like our "subnet" badge checks if the allowed routes and available routes are equal. However, it seems like the backend can return routes in an arbitrary order, which can cause the two arrays to not be equal because of sorting differences.

For example, on my local test server, depending on the order I toggle the routes in I'll get these arrays:

```
available: [ "10.0.1.0/25", "10.0.1.0/26" ]
allowed: [ "10.0.1.0/26", "10.0.1.0/25" ]
```

We'll want to sort the response we send to clients.

Thanks for reporting!

**Comment by rosszurowski** [comment]
This issue got closed because we fixed it internally. It should be corrected next time we deploy the control server (likely within 24 hours). Thanks for reporting!
